src/views/Institutions/Institutions.jsx

```diff
@@ -162,48 +190,60 @@ const Institutions = (props) => {
     }
 
     // need to remove `show` from the payload
-    let {show, ...dataToSave} = data;
-
+    let { show, ...dataToSave } = data;
     nProgress.start();
-    createInstitution(dataToSave).then(data => {
-      if (data.data.errors) {
-        setFormErrors(data.data.errors);
-      } else {
-        setAlert("Institution created successfully.", "success");
+    createInstitution(dataToSave)
```

Comment:
are we not handling Mou file upload in create institution?

Reply:
@Rajsnehil I think you missed responding to this comment. Let me know.

Reply:
@rathorevaibhav, yes we will need Mou while creating an institution.